src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/TypeLoaderEnvironment.cs

```diff
@@ -119,6 +119,8 @@ internal static void Initialize()
             Instance = new TypeLoaderEnvironment();
             RuntimeAugments.InitializeLookups(new Callbacks());
             s_nativeFormatStrings = new LowLevelDictionary<string, IntPtr>();
+            s_runtimeFieldHandles = new LowLevelDictionary<RuntimeFieldHandleKey, RuntimeFieldHandle>();
```

Comment:
Sorry I hadn't gotten a chance to respond to this yet (I checked it the changes because I got sign-off :)). The reason they need to be static is for uniqueness across the entire process, not just across an instance of the TypeLoaderEnvironment.
If we are guaranteed to have just a since instance of the TypeLoaderEnvironmnent for the entire lifetime of the process, I believe these 3 static variables can just be made instance variables. I am just not entirely sure if this is the case. Can we confirm that?